docs/meta.md

```diff
@@ -21,8 +15,9 @@ for a snap package. The following keys are mandatory:
 
 The following keys are optional:
 
-* `icon`: a SVG icon for the snap that is displayed in the store
-* `explicit-license-agreement`: set to `Y` if the user needs to accept a
+* `summary`: A short summary
+* `description`: A long description
```

Comment:
I listed it under "optional" in the docs for now.  The new "spec" does not actually specify what is optional and what is required yet. I'm happy to tweak as needed.